cmd/ebrelayer/main.go

```diff
@@ -153,6 +189,46 @@ func RunRelayerCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// RunCosmosRelayerCmd executes the initCosmosRelayerCmd with the provided parameters
+func RunCosmosRelayerCmd(cmd *cobra.Command, args []string) error {
+	// Load config file containing environment variables
+	err := godotenv.Load()
+	if err != nil {
+		return fmt.Errorf("Error loading .env file")
+	}
+
+	// Private key for validator's Ethereum address must be set as an environment variable
+	privateKey := os.Getenv("ETHEREUM_PRIVATE_KEY")
+	if strings.TrimSpace(privateKey) == "" {
+		return fmt.Errorf("Error loading validator's private key from .env file")
+	}
+
+	// Tendermint node
+	tendermintNode := args[0]
+
+	// Ethereum websocket provider
+	ethereumProvider := args[1]
+
+	// Deployed contract address
+	if !common.IsHexAddress(args[2]) {
+		return fmt.Errorf("Invalid contract address: %v", args[2])
+	}
+	contractAddress := common.HexToAddress(args[2])
+
+	// Initialize the relayer
+	err = relayer.InitCosmosRelayer(
```

Comment:
you can `return` here

Reply:
https://github.com/cosmos/peggy/pull/77/commits/be4c43930db5f73c577d2a55782bf464405dd549